packages/player/src/player/stalls.ts

```diff
@@ -59,7 +59,7 @@ async function handlePlayingEvent(
     startTimestamp,
   };
 
-  StreamingMetrics.playbackStatistics({
+  StreamingMetrics.playbackStatisticsVideo({
```

Comment:
is this change correct?

Reply:
Yeah isolated declarations require separate defs for track and video.